po/de.po

```diff
@@ -10411,15 +10410,15 @@ msgstr "Drehend"
 
 #: src/frontend.cpp:745
 msgid "Fixed"
-msgstr "Fest"
+msgstr "Unbeweglich"
```

Comment:
:x:

I doubt you can get more accurate than "Fest".

Reply:
Not sure about this. In the context of layouting on-screen elements, [I only know that "fixed" is translated as "fest"](https://wiki.selfhtml.org/wiki/HTML/Tutorials/responsive_Webdesign). Also, it might have the same and other unwanted connotations as "fest", so that I still prefer "fest".

Reply:
I suppose we would find no other German word without possible connotations. ;)
I'm fine with "Fest"